### PR TITLE
amend: respect core.commentChar

### PR DESCRIPTION
**Pull request by jerry-skydio**
Similarly we can respect the configured comment character
pretty easily by using it instead of # when applicable.

### CODE REVIEW

**Comment by jerry-skydio** [comment]
Reviews in this chain:
└https://github.com/Skydio/revup/pull/151 amend: respect git config commit.cleanup
　└**https://github.com/Skydio/revup/pull/152** **amend: respect core.commentChar**

**Comment by jerry-skydio** [comment]
| # | head | base | diff | date | summary |
| - | - | - | - | - | - |
| 0 | [333a4c46](/Skydio/revup/commit/333a4c462ca77a313e8adef97f20e60e81853a00) | [a3fc421e](/Skydio/revup/commit/a3fc421e2065425ed28f086d25e69cc914e882be) | [diff](/Skydio/revup/compare/a3fc421e2065425ed28f086d25e69cc914e882be..333a4c462ca77a313e8adef97f20e60e81853a00) | Mar 11 19:23 PM |  1 file changed, 10 insertions(+), 5 deletions(-) |

File: revup/amend.py [hunk 1]
```diff
@@ -44,13 +44,18 @@ async def invoke_editor_for_commit_msg(
...
+    comment_ret, comment_char = await git_ctx.git("config", "core.commentChar", raiseonerror=False)
+    if comment_ret != 0:
+        comment_char = "#"
+
     comments = f"{topic_summary}\n{stat_text}"
     if cleanup_type == "scissors":
         comments = f"\n{CLEANUP_SCISSOR_LINE}\n{CLEANUP_SCISSOR_COMMENT}\n{comments}"
     elif cleanup_type == "strip":
-        comments = f"\n{CLEANUP_STRIP_COMMENT}\n{comments}"
+        comments = f"\n{CLEANUP_STRIP_COMMENT.format(comment_char)}\n{comments}"
```
Comment (aaron-skydio) [inline revup/amend.py:56]: Format in an interpolated string is kinda wild, but I can't think of a nicer way lol

**Review by aaron-skydio — APPROVED** [review]
Nice, didn't even know this was a config option

**Pull request merged**